vision/google/cloud/vision/geometry.py

```diff
@@ -73,20 +87,34 @@ def __init__(self, latitude, longitude):
         self._longitude = longitude
 
     @classmethod
-    def from_api_repr(cls, response):
+    def from_api_repr(cls, location_info):
         """Factory: construct location information from Vision API response.
 
-        :type response: dict
-        :param response: Dictionary response of locations.
+        :type location_info: dict
+        :param location_info: Dictionary response of locations.
```

Comment:
I moved `latLng` back from the caller to the class.
https://github.com/GoogleCloudPlatform/google-cloud-python/pull/2918/commits/9287850457d7f8e04db2af5e7ac597eb0a8a0896